tutor/resources/styles/components/paging-navigation.less

```diff
@@ -0,0 +1,31 @@
+.tutor-paging-navigation {
+  .flex-display();
+  .justify-content(center);
+  .paging-control {
+    .flex(1);
+    .flex-display();
+    .justify-content(center);
+    .align-items(center);
+    min-height: calc(~"100vh - 150px");
+    .tutor-icon {
+      z-index: 1; //appear above content when page is extremely narrow
+      position: fixed;
+      top: 50vh;
+      color: @tutor-neutral-light;
+      font-size: 150px;
+      .transition(color 150ms linear);
+    }
+    &[disabled] {
+      cursor: not-allowed;
```

Comment:
Yeah, i kinda wondered about that.  What I was trying to prevent was confusion over "where'd the arrows go"  Originally I had them still showing, but greyed out very light.  Then I hid them all together, but neglected to remove the cursor style